src/applications/gi-sandbox/components/profile/CautionaryInformation.jsx

```diff
@@ -96,20 +100,24 @@ export function CautionaryInformation({ institution, showModal }) {
   };
 
   const allCampusesLink = (
-    <button
-      id="typeAccredited-button"
-      type="button"
-      className="va-button-link learn-more-button"
-      onClick={() => {
-        recordEvent({
-          event: 'education-navigation',
-          'edu-action': 'all-campuses',
-        });
-        showModal('allCampuses');
-      }}
-    >
-      All campuses
-    </button>
+    <div className="small-screen-font">
```

Comment:
can the `src/applications/gi-sandbox/components/LearnMoreLabel.jsx` be used here instead?

Reply:
Yes we can. I was just reusing what was there previously, but I pushed a commit using the `LearnMoreLabel` component.